docs/_pages/releases.md

```diff
@@ -7,6 +7,14 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 5.10.3
+**Fixes**
+* Fixed XPath index calculation in XML comparison when child node names are repeated in repeated parent nodes - [#1273](https://github.com/fluentassertions/fluentassertions/pull/1273)
+
+## 5.10.2
+**Fixes**
+* Added missing dependency on System.Xml - [#1248](https://github.com/fluentassertions/fluentassertions/pull/1248)
```

Comment:
Watching the overview I'm now confused why the same problem is mentioned twice!?

Reply:
Yes, there was a bug in the 5.10.1 artifact that was pushed to nuget.org.
@dennisdoomen pushed a 5.10.2 and unlisted 5.10.1
If you would just rename 5.10.1 to 5.10.2 instead, that would be perfect.